README.rst

```diff
@@ -35,6 +35,24 @@ If a user executes ``py.test`` tests with the ``--junitxml`` and with this plug-
 will contain a test suite properties element. The properties element will contain information gathered about the test
 run fetched from the local environment.
 
+Configuration
+^^^^^^^^^^^^^
+
+You can configure this plugin to tell it which 'ci-environment' you are using when you execute this test.  This will let
+pytest-rpc know what information to collect inside of the junit.xml.  If no configuration is found it will assume
+that you are using asc.
+
+1. You can configure in an ini file that is readable by pytest (setup.cfg, pytest.ini, tox.ini)::
+
+    [pytest]
+        ci-environment=mk8s
```

Comment:
This is definitely indented in raw file.

Reply:
Ok, I see what you mean now.  fIxed